[2023-04-25T00:02:36.542+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_processing.create_table scheduled__2023-04-24T00:00:00+00:00 [queued]>
[2023-04-25T00:02:36.560+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_processing.create_table scheduled__2023-04-24T00:00:00+00:00 [queued]>
[2023-04-25T00:02:36.561+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T00:02:36.561+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-04-25T00:02:36.562+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T00:02:36.574+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-04-24 00:00:00+00:00
[2023-04-25T00:02:36.581+0000] {standard_task_runner.py:55} INFO - Started process 2078 to run task
[2023-04-25T00:02:36.585+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_processing', 'create_table', 'scheduled__2023-04-24T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/database_processing.py', '--cfg-path', '/tmp/tmp_mjz1k7p']
[2023-04-25T00:02:36.589+0000] {standard_task_runner.py:83} INFO - Job 4: Subtask create_table
[2023-04-25T00:02:36.666+0000] {task_command.py:388} INFO - Running <TaskInstance: database_processing.create_table scheduled__2023-04-24T00:00:00+00:00 [running]> on host 2f9666fcabde
[2023-04-25T00:02:36.725+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-24T00:00:00+00:00
[2023-04-25T00:02:36.726+0000] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users (
                id SERIAL PRIMARY KEY,
                name VARCHAR(255) NOT NULL,
                email VARCHAR(255) NOT NULL,
                phone VARCHAR(255) NOT NULL,
                address VARCHAR(255) NOT NULL,
                organization VARCHAR(255) NOT NULL
            );
        
[2023-04-25T00:02:36.738+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-04-25T00:02:36.875+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-04-25T00:02:36.880+0000] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                id SERIAL PRIMARY KEY,
                name VARCHAR(255) NOT NULL,
                email VARCHAR(255) NOT NULL,
                phone VARCHAR(255) NOT NULL,
                address VARCHAR(255) NOT NULL,
                organization VARCHAR(255) NOT NULL
            );
        , parameters: None
[2023-04-25T00:02:36.904+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=database_processing, task_id=create_table, execution_date=20230424T000000, start_date=20230425T000236, end_date=20230425T000236
[2023-04-25T00:02:36.928+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-04-25T00:02:36.943+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
